### ISSUE TITLE
Switch to authlib

### ISSUE THREAD

**Issue by MatthewScholefield** [pyeve/flask-sentinel#35]
`flask-oautlib` has been deprecated in favor of [`authlib`](https://github.com/lepture/authlib).

**Comment by nicolaiarocci** [comment]
yes, we should move forward. Please feel free to submit a PR, I will be happy to review it.
